general/cuda.hpp

```diff
@@ -48,17 +48,40 @@
 
 // Define the MFEM inner threading macros
 #if defined(MFEM_USE_CUDA) && defined(__CUDA_ARCH__)
+#if __CUDA_ARCH__ < 600
```

Comment:
We should have a way to get rid of these atomics.

It's okay to have some kernels using them and being documented, but inside `MFEM`, we should have a way to fold back to a version not using them.

Reply:
We should get rid of this `MFEM_ATOMIC_ADD` code now that is not been used anymore.